bluesky_widgets/models/run_engine_client.py

```diff
@@ -85,6 +93,15 @@ def clear_connection_status(self):
             is_connected=self._re_manager_connected,
         )
 
+    def manager_connecting_ops(self):
+        """
+        Sequence of additional operations that should be performed while connecting to RE Manager.
+        """
+        self.load_allowed_devices()
+        self.load_allowed_plans()
+        self.load_plan_queue()
+        self.load_plan_history()
+
     def load_re_manager_status(self, *, enforce=False):
```

Comment:
A more standard name for `enforce` may be `unbuffered`. That term appears elsewhere in CPython.

Reply:
Change applied